### PR TITLE
[documentation] Update Documenter.jl

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

**Comment by tmigot** [comment]
Hi @amontoison , thank you for the PR an idea why this fais?

File: docs/make.jl [hunk 1]
```diff
@@ -3,14 +3,13 @@ using Documenter, JSOSuite
 makedocs(
   modules = [JSOSuite],
   doctest = true,
-  # linkcheck = true,
-  strict = true,
+  linkcheck = true,
```
Comment (amontoison) [inline docs/make.jl:6]:
```suggestion
  linkcheck = false,
```

**Comment by amontoison** [comment]
> Hi @amontoison , thank you for the PR an idea why this fais?

I don't understand the error so let's set `linkcheck = false`.

**Comment by abelsiqueira** [comment]
Thanks

**Pull request merged**